packages/docs/docs/cli/ffprobe.md

```diff
@@ -0,0 +1,20 @@
+---
+image: /generated/articles-docs-cli-ffprobe.png
+id: ffprobe
+title: ffprobe
```

Comment:
Title should be `npx remotion ffprobe` (consistency with other CLI command documentation pages).
Can use a `sidebar_label` to make it shorter for the sidebar.